apps/src/templates/studentSnapshot/header/index.tsx

```diff
@@ -1,27 +1,36 @@
 import {Button} from '@code-dot-org/component-library/button';
 import {SimpleDropdown} from '@code-dot-org/component-library/dropdown';
+import {Typography} from '@mui/material';
```

Comment:
This comment is not blocking, just a note.
I'd like clarification on whether we want to move to MUI or keep using our component-library.

Reply:
We are not officially moving to MUI yet, but I want to try to use as much of it as we can for this project in order to help with testing and so that I have a better idea of how it works when we eventually try to switch. 

I don't have a preference what you use, but I would like to continue to use MUI when I can. Worst case scenario, I can write a quick script that reverts everything back if MUI ends up unviable.